# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nethack/hotfixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

This fixes a few compiler and linker errors.

don't ask... please... just don't ask
blindcoder
--- ./win/Qt/qt_win.cpp	2003-06-08 20:56:11.000000000 +0200
+++ ./win/Qt/qt_win.cpp	2003-06-08 14:27:30.000000000 +0200
@@ -47,6 +47,8 @@ extern "C" {
 // enough, and we undefine NetHack macros which conflict with Qt
 // identifiers.
 
+#include <math.h>
+
 #define alloc hide_alloc // avoid treading on STL symbol
 #define lock hide_lock // avoid treading on STL symbol
 #ifdef _MSC_VER
--- ./src/Makefile 2003-06-08 20:56:11.000000000 +0200
+++ ./src/Makefile 2003-06-08 14:27:30.000000000 +0200
@@ -147,1 +147,1 @@
-	winmisc.o winstat.o wintext.o winval.o tile.o
+	winmisc.o winstat.o wintext.o winval.o
@@ -179,1 +179,1 @@
-WINQTOBJ = qt_win.o qt_clust.o qttableview.o tile.o
+WINQTOBJ = qt_win.o qt_clust.o qttableview.o
@@ -191,1 +191,1 @@
-	gnyesno.o gnworn.o tile.o
+	gnyesno.o gnworn.o
